pandas/io/parsers.py

```diff
@@ -1671,7 +1671,8 @@ def __init__(self, src, **kwds):
 
         ParserBase.__init__(self, kwds)
 
-        if 'utf-16' in (kwds.get('encoding') or ''):
+        if kwds.get('compression') is None \
```

Comment:
comment here

Reply:
Also, use parentheses instead of the backslash.